pages/about.tsx

```diff
@@ -25,47 +25,50 @@ export default function AboutPage() {
   return (
     <>
       <IndividualPageHead
-        title="The story and team behind Clearviction."
-        metaContent="We're empowering individuals with criminal records in Washington State, and building brighter futures with tools like our Eligibility Calculator."
+        title={content.meta.title}
+        metaContent={content.meta.content}
       />
 
       <HeroBanner
-        header={aboutContent.hero.header}
-        subheading={aboutContent.hero.body}
-        subheading2={aboutContent.hero.subheading2}
-        imgsrc={aboutContent.hero.imgsrc}
+        header={content.hero.header}
+        subheading={content.hero.body}
+        subheading2={content.hero.subheading2}
+        imgsrc={content.hero.imgsrc}
       />
+
       <SectionContainer>
         <ButtonGroup
           variant="text"
           fullWidth
           orientation={matchesXS ? 'vertical' : 'horizontal'}
         >
-          <Button href="#our-story">Our Story</Button>
-          <Button href="#our-team">Our Team</Button>
+          <Button href="#our-story">{content.buttons[0].name}</Button>
+          <Button href="#our-team">{content.buttons[1].name}</Button>
```

Comment:
It looks like the hrefs are in the content file along with the text, can we change the href here to use those?